doc/item_bonus.txt

```diff
@@ -198,6 +198,14 @@ bonus2 bSkillUseSPrate,sk,n;		Decreases SP consumption of skill sk by n%
 Atk/Def
 -------
 bonus2 bSkillAtk,sk,n;    		Increases damage of skill sk by n%
+bonus bSkillRatio,n;			Adds n to the skillratio of all attacks/skills
+					Detailed Explanation (all examples assume n=5):
+					- Normal attacks will have their skillratio increased (100% -> 105%)
+					- Only skills that have a skill ratio are affected, skills that deal fixed damage are not affected
+					- Both physical and magical skills are affected
+					- Bonus benefits skills with low skill ratio more (200% -> 205%, 800% -> 805%)
+					- Bonus is applied to each hit of skills that multiply damage by number of hits (100%*8 -> 105%*8)
+					- This bonus works the same way as status changes like Provoke in renewal and stacks linearly with them
```

Comment:
```suggestion
bonus bSkillRatio,n;			Adds n to the skillratio of all attacks/skills
```

Reply:
I also prefer single line explanation (see my original version), but Atemo requested more information to be added.

Reply:
I think we should keep it the way you originally had it.
If we want to add more information to the specific bonus, then we have to rethink/redesign the whole file.
We can discuss that, if we think this would be useful.

@Atemo is that okay for you?

Reply:
Yeah it's okay